.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Code of conduct
+
+This project operates under the [Apereo Foundation Welcoming Policy][].
```

Comment:
At tremendous risk of being obnoxiously pedantic, but also in the hope of avoiding the cardinal sin of misleading potential vulnerable contributors as to the degree of and substance behind commitment to the code of conduct,

Where we're at in https://github.com/ChristianMurphy/apereo.foundation/issues/9#issuecomment-406333861 is:

> Apereo is soon going to start a communications review process, including the website.
This redesign may be included in the feedback and options for that review.

Which means

+ Apereo has not yet started communications review process
+ this redesign may not be included in the options for that review

which makes the status quo relationship to Apereo, well, tenuous. Is it really true that Apereo is prepared to field a Code of Conduct issue arising from this project? Or is the reality of where we're at that the relationship of this project to Apereo the corporation is not yet sufficiently clear for this project to rely upon the Apereo Welcoming Policy?

I feel differently about the nuances of referencing the Welcoming Policy from sponsored projects, or incubating projects, or even contrib ancillary projects in the context of sponsored or contrib primary projects, or repos in the Apereo organization, or even repos in the ancillary organization... but this is none of those things currently. It's had no explicit mention on e.g. `open@` so it's not entirely clear to me how participants in Apereo projects would know this initiative were afoot.

So. Are we at a point where we can in good faith say this project operates under the Apereo Welcoming Policy, or would it be better to be more conservative about this and say that we aspire to do so but in the meantime the project participants intend to operate _as if_ the policy were effective and here's now to get in touch?

How about 

```
This project aspires to be an Apereo project and therefore to operate under the [Apereo Foundation Welcoming Policy][]. 

We're still working out governance and formal relationship to Apereo the corporation ( #9 ). 
In the meantime we intend to conduct this project and care for its participants _as if_ this project were 
unambiguously subject to the Apereo Welcoming Policy. 

Got concerns of any kind? Get in touch with project founders @ChristianMurphy and/or @apetro.
```

Reply:
> Is it really true that Apereo is prepared to field a Code of Conduct issue arising from this project?

Not entirely fair.
This question could be shortened to:

> Is it really true that Apereo is prepared to field a Code of Conduct issue?

and still be tenuous given the response to https://groups.google.com/a/apereo.org/forum/#!msg/open/PU2vtcsJmCs/THUn5GyMAgAJ

---

> Or is the reality of where we're at that the relationship of this project to Apereo the corporation is not yet sufficiently clear for this project to rely upon the Apereo Welcoming Policy?

:man_shrugging: Apereo leans heavily on Apache bylaw, I don't see this as being all that different.

---

> It's had no explicit mention on e.g. open@

A post is forth coming

---

> it's not entirely clear to me how participants in Apereo projects would know this initiative were afoot

Invites have gone out to core members of as many as I can track down committer lists for (sorry to folks who haven't got invites yet, please document the project committers, and update the link in https://github.com/ChristianMurphy/apereo.foundation/blob/master/.github/COMMITTERS.md)

---

```
This project aspires to be an Apereo project and therefore to operate under the [Apereo Foundation Welcoming Policy][]. 

We're still working out governance and formal relationship to Apereo the corporation ( #9 ). 
In the meantime we intend to conduct this project and care for its participants _as if_ this project were 
unambiguously subject to the Apereo Welcoming Policy. 

Got concerns of any kind? Get in touch with project founders @ChristianMurphy and/or @apetro.
```

:man_shrugging: I guess, it feels a bit watered down.
If we need something that can operate independent of Apereo at first, I'd tend to lean toward.
https://www.contributor-covenant.org/version/1/4/code-of-conduct